sdk/monitor/azure-monitor-query/README.md

```diff
@@ -90,7 +90,127 @@ LogsAsyncClient logsAsyncClient = new LogsClientBuilder()
                         + "; value = " + logsTableCell.getValueAsString()));
         }
     }
-}
+
+```
+### Get logs for a query and read the response as a model type
+
+```java
+
+    LogsQueryResult queryResults = logsClient
+        .queryLogs("d2d0e126-fa1e-4b0a-b647-250cdd471e68", "AppRequests", null);
+    
+    // Sample to use a model type to read the results
+    for (LogsTable table : queryResults.getLogsTables()) {
+      for (LogsTableRow row : table.getTableRows()) {
```

Comment:
I was thinking about our conversation in java sync up - and was not very clear on how to use custommodel with only some fields defined.  In fact, I could not find anything in Java that is like the dynamic objects of c#. I will ping you offline to know more about this :)